presto-tpcds/src/main/java/com/facebook/presto/tpcds/TpcdsRecordSet.java

```diff
@@ -57,7 +57,7 @@ public TpcdsRecordSet(Results results, List<Column> columns)
         this.columns = ImmutableList.copyOf(columns);
         ImmutableList.Builder<Type> columnTypes = ImmutableList.builder();
         for (Column column : columns) {
-            columnTypes.add(getPrestoType(column.getType()));
+            columnTypes.add(getPrestoType(column.getType(), false));
```

Comment:
Please add a comment that this code is only used by the Java connector which doesn't allow this option.